jenkins/osa_onmetal.functions.groovy

```diff
@@ -374,7 +374,7 @@ def setup_parse_persistent_resources(){
     
     sh """
     ssh -o StrictHostKeyChecking=no root@${host_ip} '''
-    git clone https://github.com/osic/persistent-resources-tests-parse.git
+    git clone https://github.com/osic/persistent-resources-tests-parse.git /root/persistent-resources-tests-parse
     pip install -e /root/persistent-resources-tests-parse
```

Comment:
Your setup.py is wong, you are not specifying there what files/packages you want to include in your package. Add that to your setup.py or setup.cfg and that should work. If you have questions let me know and I can help, but we cannot integrate this in dev mode.